api/fittings/lisk_response_formatter.js

```diff
@@ -45,7 +45,11 @@ module.exports = function create() {
 
 		if (_.isEmpty(context.input)) {
 			context.headers = { 'content-type': 'application/json' };
-			next(null, {});
+			next(null, {
+				meta: {},
+				data: context.input,
```

Comment:
W can skip `data: context.input` by just `data: {}` as we know `_.isEmpty(context.input)`

Reply:
`context.input` can be an empty object or an empty array, so I think its better that we keep `context.input` rather than defaulting it to empty object `data: {}`